assets/src/dashboard/app/views/editorSettings/adManagement/index.js

```diff
@@ -45,6 +46,18 @@ export const TEXT = {
     'https://amp.dev/documentation/guides-and-tutorials/develop/advertise_amp_stories/',
     'web-stories'
   ),
+  SITE_KIT_NOT_INSTALLED: __(
+    'Install <a>Site Kit by Google</a> to easily enable Google AdSense for Web Stories.',
+    'web-stories'
+  ),
+  SITE_KIT_INSTALLED: __(
+    'Use Site Kit by Google to easily<a>activate Google AdSense</a> for Web Stories.',
+    'web-stories'
+  ),
+  SITE_KIT_IN_USE: __(
+    'Site Kit by Google has already enabled Google AdSense for your Web Stories, all changes to your ad configuration should occur there.',
+    'web-stories'
+  ),
```

Comment:
💭 These already exist here: https://github.com/google/web-stories-wp/blob/main/assets/src/dashboard/app/views/editorSettings/googleAnalytics/index.js#L58-L68. Wondering if we should just import those to not duplicate these?

Reply:
Those strings refer to Google Analytics though, whereas the strings here refer to Google AdSense.

Reply:
@samwhale As Pascal notes, those strings have words google analytics in them. These say  Google AdSense. Sadly I just had to copy and paste, then change the strings.